packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -103,6 +108,58 @@ const createDatetimeDisplays = (compareDatetimeDisplays) => {
   return null;
 };
 
+const createSystolicDisplay = (systolic, hideUnit, diastolicUnit, id) => {
+  if (!Object.prototype.hasOwnProperty.call(systolic, 'noData')) {
+    return <ResultError key={`Error-Systolic-${id}`} />;
+  }
+  if (systolic.noData) {
+    return <NoData key={`NoData-Systolic-${id}`} />;
+  }
+  const systolicDisplay = (
+    <ConditionalWrapper
+      key={`del-Systolic-${systolic.eventId}`}
+      condition={systolic.statusInError}
+      wrapper={children => <del>{children}</del>}
+    >
+      <Observation
+        key={`Observation-Systolic-${systolic.eventId}`}
+        eventId={systolic.eventId}
+        result={systolic.result}
+        interpretation={!systolic.statusInError ? systolic.interpretation : null}
+        isUnverified={systolic.isUnverified}
+        hideUnit={hideUnit || ((systolic.cleanedUnit === diastolicUnit) && !systolic.statusInError)}
+      />
+    </ConditionalWrapper>
+  );
+  return systolicDisplay;
+};
+
+const createDiastolicDisplay = (diastolic, hideUnit, id) => {
+  if (!Object.prototype.hasOwnProperty.call(diastolic, 'noData')) {
+    return <ResultError key={`Error-Diastolic-${id}`} />;
+  }
+  if (diastolic.noData) {
+    return <NoData key={`NoData-Diastolic-${id}`} />;
+  }
+  const diastolicDisplay = (
+    <ConditionalWrapper
+      key={`del-Diastolic-${diastolic.eventId}`}
+      condition={diastolic.statusInError}
+      wrapper={children => <del>{children}</del>}
+    >
+      <Observation
+        key={`Observation-Diastolic-${diastolic.eventId}`}
+        eventId={diastolic.eventId}
+        result={diastolic.result}
+        interpretation={!diastolic.statusInError ? diastolic.interpretation : null}
+        isUnverified={diastolic.isUnverified}
+        hideUnit={hideUnit}
```

Comment:
From what I can tell, it seems the biggest differences between `createSystolicDisplay` and `createDiastolicDisplay` are
- the use of the word `Systolic ` vs `Diastolic` in the key names
- how the hideUnit value is handled. 

1. could we combine these to be resuable and take a `type` parameter?
2. would it make sense to break his out into it's own class so we can validate the rendered component in testing? There are ~5-6 combinations that affect how the Systolic result  renders and 4 combos rendering variants for diastolic result.

Reply:
Made _BloodPressureDisplay component https://github.com/cerner/terra-clinical/pull/648/commits/bcd9623974696cc69c750cf77514f41a424ac509#diff-db5d557fed702f3b0bcac70aa9a018eaR9